structuredefinitions/Extension-England-RemovalReason.xml

```diff
@@ -0,0 +1,61 @@
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="Extension-England-RemovalReason" />
```

Comment:
Need to be Extension-England-FGMRemovalReason

Reply:
Agree. Please modify